### ISSUE TITLE
Automatic watchers

### ISSUE THREAD

**Issue by JerryDoubleU** [xripcsu/rxjs-watcher#3]
It would be great if this tool could automatically watch observables, so we could just plug a module into the app and then filter out interesting streams through the DevTools.

**Comment by JerryDoubleU** [comment]
Something like Redux DevTools does for the NgRx - that would be a killer feature!

**Comment by xripcsu** [comment]
I think that it would be hard to distinguish between different observables 

for example 
```ts 
source1.pipe(
  filter(),
  map()
).subscribe()

source2.pipe(
  switchmap(() => source3),
  filter()
).subscribe()
```
We would need to show 6 marble diagrams for source1 and source2 and also on every emitted value from source2 it needs to create new marble diagram. We would not have any label for them so it would be very hard to find out what we want.

**Comment by JerryDoubleU** [comment]
I agree, this is a complicated thing. I'm not familiar with the capabilities of Chrome DevTools plugin, but I have an idea, that a list would get created and you could select the streams that you're interested in labeled by the filename and line in the source code.

**Comment by chriszrc** [comment]
I wonder if this would be possible to achieve with a decorator that could take an optional name, e.g `@Watch('source1')`. It would be really great to be able to see the rxJS actions without having to make actual code modifications. The decorators could stay in place and maybe just be turned off in production
